apiserver/Volumes.md

```diff
@@ -7,96 +7,81 @@ API server allows to specify multiple types of volumes mounted to the Ray pods (
 [config maps](https://kubernetes.io/docs/concepts/storage/volumes/#configmap),
 [secrets](https://kubernetes.io/docs/concepts/storage/volumes/#secret),
 and [empty dir](https://kubernetes.io/docs/concepts/storage/volumes/#emptydir).
-Multiple volumes of different type can be mounted to both head and worker nodes, by defining a volume array for them 
```

Comment:
This file is already updated in https://github.com/ray-project/kuberay/pull/1561 with even more changes. Can you remove it from this PR?